src/components/Talk/TalkEditor.vue

```diff
@@ -408,7 +407,7 @@ const duplicateAudioItem = async () => {
   const prevAudioItem = store.state.audioItems[prevAudioKey];
```

Comment:
（ちゃんと理解できてないのですが）
おそらくVuex内の`state`はどうやら`ref`になっているらしく、`toRaw`が必要そうでした！
```suggestion
  const prevAudioItem = toRaw(store.state.audioItems[prevAudioKey]);
```